--- a/examples/SplashCursor.tsx
+++ b/examples/SplashCursor.tsx
@@ -1,6 +1,6 @@
 "use client";
 import { useEffect, useRef } from "react";
 
-function SplashCursor({
+export default function SplashCursor({
   // Add whatever props you like for customization
   SIM_RESOLUTION = 128,
   DYE_RESOLUTION = 1440,
@@ -1082,7 +1082,7 @@
     }
 
     function HSVtoRGB(h, s, v) {
-      let r, g, b, i, f, p, q, t;
+      const r, g, b, i, f, p, q, t;
       i = Math.floor(h * 6);
       f = h * 6 - i;
       p = v * (1 - s);
